operator/controllers/kyma_controller_remote_sync_test.go

```diff
@@ -47,28 +48,29 @@ var _ = Describe("Kyma with multiple module CRs in remote sync mode", Ordered, f
 
 	It("CR add from client should be synced in both clusters", func() {
 		By("Remote Kyma created")
-		Eventually(RemoteKymaExists(runtimeClient, kyma.GetName()), 30*time.Second, Interval).Should(Succeed())
-		remoteKyma, err := GetKyma(runtimeClient, kyma.GetName())
+		Eventually(RemoteKymaExists(runtimeClient, kyma.GetName()), 30*time.Second, testhelper.Interval).Should(Succeed())
+		remoteKyma, err := testhelper.GetKyma(ctx, runtimeClient, kyma.GetName())
 		Expect(err).ShouldNot(HaveOccurred())
 
 		By("add skr-module-client to remoteKyma.spec.modules")
 		remoteKyma.Spec.Modules = []v1alpha1.Module{
 			*skrModuleFromClient,
 		}
-		Eventually(runtimeClient.Update(ctx, remoteKyma), Timeout, Interval).Should(Succeed())
+		Eventually(runtimeClient.Update(ctx, remoteKyma), testhelper.Timeout, testhelper.Interval).Should(Succeed())
 
 		By("skr-module-client created in kcp")
-		Eventually(ModuleExists(kyma.GetName(), skrModuleFromClient.Name), Timeout, Interval).Should(BeTrue())
+		Eventually(ModuleExists(kyma.GetName(), skrModuleFromClient.Name),
+			testhelper.Timeout, testhelper.Interval).Should(BeTrue())
 	})
 })
 
 var _ = Describe("Kyma sync into Remote Cluster", Ordered, func() {
-	kyma := NewTestKyma("kyma-test-remote-skr")
+	kyma := testhelper.NewTestKyma("kyma-test-remote-skr")
```

Comment:
I think you could use a "." import for testhelper so you dont have tos specify it for every call.